server/src/main/java/com/paypal/selion/utils/SauceConfigReader.java

```diff
@@ -102,10 +115,24 @@ public void invalidate() {
         SauceConfigReaderHolder.invalidate();
     }
 
+    private void restoreDefaults() {
+        sauceRetry = DEFAULT_RETRY_COUNT;
+        sauceTimeout = DEFAULT_TIMEOUT;
+        sauceURL = "";
+        authKey = "";
+        defaultParentTunnel = DEFAULT_TUNNEL;
+        defaultTunnelIdentifier = DEFAULT_TUNNEL;
+        requireUserCredentials = false;
+    }
+
     /**
      * Load the all the properties from JSON file(sauceConfig.json)
      */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method loadConfig() has an NPath complexity of 217](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2439971443/issues/source?bid=2928425&fileBranchId=3233060#l131)